projects/igniteui-angular/src/lib/progressbar/progressbar.component.ts

```diff
@@ -203,27 +196,48 @@ export abstract class BaseProgressDirective {
         return this._max;
     }
 
+    @HostBinding('style')
```

Comment:
This means that any style attribute the end-users apply to the progressbar component will be overridden by this method.